hudi-flink/src/main/java/org/apache/hudi/streamer/FlinkStreamerConfig.java

```diff
@@ -287,7 +283,6 @@
     org.apache.flink.configuration.Configuration conf = fromMap(propsMap);
 
     conf.setString(FlinkOptions.PATH, config.targetBasePath);
-    conf.setString(FlinkOptions.READ_AVRO_SCHEMA_PATH, config.readSchemaFilePath);
```

Comment:
Can we also refactor the `READ_AVRO_SCHEMA_PATH ` config option. I would like to remove `READ_`. WDYT?

Reply:
> Can we also refactor the `READ_AVRO_SCHEMA_PATH ` config option. I would like to remove `READ_`. WDYT?

Maybe `SOURCE_` is more appropriate.

we add the prefix string is because hudi supports schema evolution. in `hudi-utilities` module we call it sourceSchema and targetSchema, these two schemas may be different.

here Danny calls it `READ_`, that's because it is used in streaming read too, also reasonable.

I actually prefer `SOURCE_` and `TARGET_` prefix, taking schema evolution support into consideration

Reply:
s.g.

Reply:
@danny0405 Do you agree to rename `READ_AVRO_SCHEMA_PATH` to `SOURCE_AVRO_SCHEMA_PATH`?